src/main/kotlin/com/launchdarkly/intellij/settings/LaunchDarklyApplicationSettings.kt

```diff
@@ -263,7 +263,7 @@ class LaunchDarklyApplicationConfigurable : BoundConfigurable(displayName = "Lau
 
         try {
             environmentContainer = getEnvironmentContainer(projectBox.selectedItem.toString())
-            val envMap = environmentContainer.environments.map { it.key }.sorted()
+            val envMap = environmentContainer.environments.map { formatKeyAndName(it.key, it.name) }.sorted()
```

Comment:
from Darrin's feedback on slack, we don't need this for environment for now

Reply:
Removed format method from environment